libs/expo/shared/ui-components/src/lib/Avatar/Avatar.tsx

```diff
@@ -44,15 +46,17 @@ export function Avatar(props: IAvatarProps) {
     accessibilityHint,
     hasBorder,
     borderColor,
+    loading,
   } = props;
 
-  const getTextComponent = (size: 'sm' | 'md' | 'lg') => {
+  const getTextComponent = (size: 'sm' | 'md' | 'lg' | 'xl') => {
     switch (size) {
       case 'sm':
         return <UserOutlineIcon size="sm" color={Colors.PRIMARY_EXTRA_DARK} />;
       case 'md':
+        return <UserOutlineIcon size="lg" color={Colors.PRIMARY_EXTRA_DARK} />;
       case 'lg':
-        return <UserOutlineIcon color={Colors.PRIMARY_EXTRA_DARK} />;
+        return <UserOutlineIcon size="xl" color={Colors.PRIMARY_EXTRA_DARK} />;
```

Comment:
was `xl` meant to be be under `case "lg"`?

Reply:
fixed everywhere